src/pages/Login/SignIn/SignIn.tsx

```diff
@@ -60,6 +64,7 @@ export const SignIn: FC = () => {
             setPassword(value);
           }
         }}
+        isSmall={isSmallMobile}
```

Comment:
Worth moving this functionality into the component itself? So it is applied for all inputs?